pkgs/by-name/hy/hyprviz/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  gtk4,
+  glib,
+  pango,
+  pkg-config,
+  wrapGAppsHook4,
+  hyprland,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
a bit more convenient for recursion and referencing attrs (also allows you referencing implicitly set attrs)